frontend/src/pages/projects/screens/detail/storage/ManageStorageModal.tsx

```diff
@@ -84,7 +87,9 @@ const ManageStorageModal: React.FC<AddStorageModalProps> = ({ existingData, isOp
       }
       return;
     }
-    const createdPvc = await createPvc(createData, namespace, { dryRun });
+    const createdPvc = await createPvc(createData, namespace, storageClassName?.metadata.name, {
```

Comment:
if we end up renaming the const, we would then have to use i.e.

` const createdPvc = await createPvc(createData, namespace, storageClassPreferred?.metadata.name, {`